lib/dbservice_web/plugs/authentication_middleware_plug.ex

```diff
@@ -35,7 +35,7 @@ defmodule DbserviceWeb.AuthenticationMiddleware do
     end
   end
 
-  defp is_swagger_request?(referer, request_path) do
+  defp swagger_request?(referer, request_path) do
```

Comment:
Can you double check this function name? In my PR, this was the issue, i renamed it to `contain_swagger_request`.

Reply:
okay, let it be `contain_swagger_request`